app/javascript/utilities/documentation/index.js

```diff
@@ -0,0 +1,160 @@
+const fs = require('fs');
```

Comment:
This code is not for the app, but for tooling, so still not sure if it belongs in this folder. To be clear though, it won't get bundled via webpack or sprockets.

Reply:
@rhymes @citizen428, maybe `bin/utitilies`?

Reply:
This may not be the most popular opinion, but unless you use this outside `bin/generate-css-utility-classes-docs.js` too, I'd probably just move it in there. I'd rather have everything in the same place if there's no reuse anyway, and that we're not quite sure where to put this makes me think that even more.

Reply:
Yeah, sounds good. Some stuff could potentially be reused, but I'm not going to YAGNI. I'll just rework things a bit so that it can still be tested. It's not production code, but it does affect the output of Storybook.